### PR TITLE
Include experimental data and literature link. 

### PR DESCRIPTION
**Pull request by amezcum1**
Adds CB8 experimental data (as provided) and the new publication link for now. Updates to README files.

### CODE REVIEW

File: host_guest/Analysis/ExperimentalMeasurements/README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+## Manifest
+
+- `experimental_measurements.pdf`: Summary table of the experimental data after conversion
+- `experimental_measurements.csv`: Summary table of the experimental data after conversion, in `.csv` format
```
Comment (davidlmobley) [inline host_guest/Analysis/ExperimentalMeasurements/README.md:4]: Not seeing the .csv, .json etc here yet.

**Review by davidlmobley — CHANGES_REQUESTED** [review]
Looks good except I don't thinkyou have the machine-readable stuff here yet, though the README says it is.

File: host_guest/Analysis/ExperimentalMeasurements/README.md (hunk 1 above)
Comment (amezcum1) [reply host_guest/Analysis/ExperimentalMeasurements/README.md]: Removed this for now.

**Comment by davidlmobley** [comment]
Thanks!

**Pull request merged**